src/button.js

```diff
@@ -1,7 +1,7 @@
 import React, {Component} from 'react'
 import PropTypes from 'prop-types'
 
-import {AUTOCOMPLETE_CONTEXT} from './constants'
+import AUTOCOMPLETE_CONTEXT from './context'
```

Comment:
Let's leave it as `constants`, just in case we want to have more in the future for some reason. I see no real reason to change it.